<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWindow</class>
<!-- main widget -->
 <widget class="DepositWindow" name="DepositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Депозитный калькулятор</string>
  </property>

  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>закрыть</string>
   </property>
  </widget>

  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
<!-- input data -->
  <widget class="QLabel" name="deposit_sum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sum">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="interest_rate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_interest">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="tax_rate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Налоговая ставка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tax_rate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="payments_period">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок вклада</string>
   </property>
  </widget>
    <widget class="QLineEdit" name="payments_period_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>95</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="payments_period_choice">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>130</y>
     <width>95</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>День</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Неделя</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Год</string>
    </property>
   </item>
  </widget>

<!-- radio button for capitalisation -->
   <widget class="QRadioButton" name="radioButton_capitalize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Капитализация процентов</string>
    </property>
   </widget>

<!-- output labels -->
  <widget class="QLabel" name="monthly_payment_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Начислено процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="input_percents">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Начислено налогов</string>
   </property>
  </widget>
  <widget class="QLabel" name="input_taxes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Итоговая сумма</string>
   </property>
  </widget>

<!-- outputs -->
<widget class="QLabel" name="output_percents">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
  <widget class="QLabel" name="output_taxes">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
  <widget class="QLabel" name="output_end_sum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>


 </widget>

 <customwidgets>
  <customwidget>
   <class>DepositWindow</class>
   <extends>QWidget</extends>
   <header>depositwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
